### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

### CODE REVIEW

File: src/event-tracker.ts [hunk 1]
```diff
@@ -92,10 +92,8 @@ export class EventTracker {
 		(event: string) => {
 			// Call this function at maximum once every minute
 			return _.throttle(
-				properties => {
-					if (this.client != null) {
-						this.client.track(event, properties);
-					}
+				(properties: EventTrackProperties | Error) => {
```
Comment (Page-) [inline src/event-tracker.ts:95]: Updated typescript picked up that this was an implicit any

File: src/event-tracker.ts [hunk 2]
```diff
@@ -92,10 +92,8 @@ export class EventTracker {
...
+					this.client?.track(event, properties);
```
Comment (Page-) [inline src/event-tracker.ts:96]: This change wasn't strictly necessary but is part of incremental improvement

File: src/compose/network.ts [hunk 3]
```diff
@@ -217,7 +217,9 @@ export class Network {
 	}
 
 	private static validateComposeConfig(
-		config: Partial<ComposeNetworkConfig>,
+		config: Partial<Omit<ComposeNetworkConfig, 'ipam'>> & {
+			ipam?: Partial<ComposeNetworkConfig['ipam']>;
+		},
```
Comment (Page-) [inline src/compose/network.ts:222]: Updated typescript picked up that this is the actual type we call this with so I had to amend it here to accept it

**Comment by 20k-ultra** [comment]
Do we want to include updating modules that contain fixes for vulnerabilities via `npm audit fix` in this PR?

**Comment by 20k-ultra** [comment]
I've fixed the conflicts with master that the current commit had. I'll be adding another commit with the updates from `npm audit fix` so that we can revert separately if there are any issues.

**Comment by 20k-ultra** [comment]
Ready for review @Page- @CameronDiver

**Comment by CameronDiver** [comment]
@balena-ci rebase

**Pull request merged**